wavefront/server/apps/call_processing/call_processing/services/language_detection_tool.py

```
@@ -0,0 +1,199 @@
+                        return
+
+                    # Append new language instruction to clean base prompt
+                    updated_content = f'{base_prompt}\n\n{language_instruction}'
+                    updated_system_message = {
+                        'role': 'system',
+                        'content': updated_content,
+                    }
+
+                    # Update context
+                    current_messages = context.get_messages()
+                    new_messages = [updated_system_message] + current_messages[1:]
```

Ruff (0.14.14)

[warning] Consider `[updated_system_message, *current_messages[1:]]` instead of concatenation

Replace with `[updated_system_message, *current_messages[1:]]`

(RUF005)